package.json

```
@@ -118,7 +106,7 @@
   "main": "./index.js",
   "types": "./types/index.d.ts",
   "engines": {
-    "node": ">=16.20.1"
+    "node": ">=18.0.0"
```

```suggestion
    "node": ">=20.19.0"
```

As of [`mongodb@7.0.0`, it requires `>=20.19.0`](https://app.unpkg.com/mongodb@7.0.0/files/package.json#L117)